src/test/java/org/jenkinsci/plugins/workflow/multibranch/JobPropertyStepTest.java

```diff
@@ -336,12 +337,86 @@ public void resetStartsAndStops() {
         assertTrue(p.getTriggers().isEmpty());
 
         // Now add a trigger.
+        // Looking for core versions 2.21 and later for the proper pollScm symbol, rather than the broken scm symbol.
+        if (SCMTrigger.DescriptorImpl.class.isAnnotationPresent(Symbol.class)
+                && SymbolLookup.getSymbolValue(SCMTrigger.DescriptorImpl.class).iterator().next().equals("pollScm")) {
+
+            p.setDefinition(new CpsFlowDefinition(
+                    "properties([pipelineTriggers([\n"
+                            + "  pollScm(scmpoll_spec: '@daily', ignorePostCommitHooks: true), [$class: 'MockTrigger']])])\n"
```

Comment:
We can! See the test below, where we are - I wanted to cover both the `pollSCM('@daily')` case and where you want to specify `ignorePostCommitHooks`.